### ISSUE TITLE
Error message if google maps api is not loaded

### ISSUE THREAD

**Issue by easyest** [jawj/OverlappingMarkerSpiderfier#75]
There should be a message if OMS is loaded without google maps api. ATM the script silently dies.

**Comment by TheTerrasque** [comment]
Sweet Jesus! Thank you, I finally figured out why the code wouldn't work!

Not giving any error and just die silently is a terrible design.

**Comment by easyest** [comment]
It seems the project is long dead...

**Comment by jawj** [comment]
Nope, I'm just really busy! I'll certainly consider a pull request on this one.
